msdk-datamodel/src/main/java/io/github/msdk/datamodel/impl/SimpleChromatogramDataPointList.java

```diff
@@ -124,7 +124,7 @@ public void setBuffers(@Nonnull ChromatographyInfo[] rtBuffer,
 
         // Check if the RT array is properly sorted
         for (int pos = 1; pos < newSize; pos++) {
-            if (rtBuffer[pos].compareTo(rtBuffer[pos - 1]) >= 0)
+            if (rtBuffer[pos - 1].compareTo(rtBuffer[pos]) >= 0)
                 throw (new MSDKRuntimeException(
```

Comment:
I agree. Also, the comparison should be > and not >=.
(Alternatively, you can use the previous comparison and change the operator to < )